### PR TITLE
Add alert MDX custom component

### PR DESCRIPTION
**Pull request by impurity-dev**
image
The image shows the original warning, and the bottom shows the use of the @mui/material Alert component. I have added the custom <Alert ...> markdown to be used anywhere.

I did _not_ replace all instances of the alerts, as I wanted to ensure this custom component was good to go first. I did replace the glTF.md warning, as an example. If this merges, I plan to go through and update all alert instances as a follow-up.

Issue: #207

### CODE REVIEW

File: yarn.lock
Comment (RaananW) [inline yarn.lock:1]: if no dependency was updated, it would be wonderful to revert this change

**Review by RaananW — APPROVED** [review]
This is GREAT!! thanks a lot for that. Just a small comment about the lock file

File: yarn.lock
Comment (impurity-dev) [reply yarn.lock:1]: @RaananW No worries at all. Reverted 😄
